src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -10,100 +15,151 @@ const StemDeniedDetails = ({
-          You don’t meet the benefit requirements for the Rogers STEM
-          Scholarship
-          <ul className="stem-ad-list-secondary">
-            <li className="stem-ad-list-item">
-              By law, all Rogers STEM Scholarship recipients must have 6 months
-              or less of Post 9/11 GI Bill (Chapter 33) benefits left (38 U.S.
-              Code § 3320).
-            </li>
+        {remainingEntitlement > 180 && (
+          <li className="stem-ad-list-item">
+            You didn’t meet the benefit requirements for the Rogers STEM
+            Scholarship
+            <ul className="stem-ad-list-secondary">
+              <li className="stem-ad-list-item">
+                According to your service and school data records on file, as of{' '}
+                {date}, you have more than 6 months of Post-9/11 GI Bill
+                (Chapter 33) benefits remaining.
+              </li>
+              <li className="stem-ad-list-item">
+                By law, all Rogers STEM Scholarship recipients must have 6
+                months or less of Post-9/11 GI Bill (Chapter 33) benefits left
+                (38 U.S. Code § 3320).
+              </li>
+            </ul>
+          </li>
+        )}
+        {isEnrolledStem === false &&
+          isPursuingTeachingCert === false && (
             <li className="stem-ad-list-item">
-              We reviewed your service and school data records currently on
-              file. As of [{date}
-              ], you have more than 6 months of Post 9/11 GI Bill (Chapter 33)
-              benefits remaining.
+              You didn’t meet the degree requirements for the Rogers STEM
+              Scholarship because you answered “No” to both of the following
+              questions on your application. To be eligible, you must answer
+              “Yes” to one of these questions.
+              <ul className="stem-ad-list-secondary">
+                <li className="stem-ad-list-item">
+                  "Are you enrolled in a science, technology, engineering, or
+                  math (STEM) degree program?"
+                </li>
+                <li className="stem-ad-list-item">
+                  "Do you have a STEM undergraduate degree and are now pursuing
+                  a teaching certification?"
+                </li>
+                <li className="stem-ad-list-item">
+                  By law, all Rogers STEM Scholarship recipients must either:
+                  <ol>
+                    <li>
+                      Be enrolled in an eligible STEM undergraduate degree,{' '}
+                      <strong>or</strong>
+                    </li>
+                    <li>
+                      Have an eligible STEM undergraduate degree and be pursuing
+                      a teaching degree
+                    </li>
+                  </ol>
+                </li>
+              </ul>
             </li>
-          </ul>
-        </li>
+          )}
```

Comment:
We still don't want this as a reason for denial. There is a possibility they could answer both questions no and still be eligible due to another criteria to be added later. The conditional for the affirmative is ok.